js/ui/grid_core/ui.grid_core.column_fixing.js

```diff
@@ -762,7 +762,7 @@ var RowsViewFixedColumnsExtender = extend({}, baseFixedColumns, {
             columns,
             cellIndex = 0;
 
-        if($fixedTable && $fixedTable.find($cell).length) {
+        if($fixedTable && $fixedTable.find($cell).length && $cell.is("td")) {
```

Comment:
Maybe change order:
$cell.is("td") && $fixedTable && $fixedTable.find($cell).length